src/Model/Isodose.py

```diff
@@ -95,4 +95,26 @@ def get_dose_grid(rtd, z=0):
 
             return plane
 
-        return np.array([])
\ No newline at end of file
+        return np.array([])
+
+
+def calculate_rxdose(rtplan):
+    rxdose = 1
+    if ('DoseReferenceSequence' in rtplan and
+            rtplan.DoseReferenceSequence[0].DoseReferenceStructureType and
+            rtplan.DoseReferenceSequence[0].TargetPrescriptionDose):
+        rxdose = rtplan.DoseReferenceSequence[0].TargetPrescriptionDose * 100
```

Comment:
Magic Number.
CGRAY_PER_GRAY = 100 
or 
GRAY_TO_CGRAY_SCALE_FACTOR = 100
which also suggests that if someone writes this out to a DICOM object somewhere, they had better get the scaling right or you've just killed someone.
I would refactor by renaming the variable from "rxdose" to "rx_dose_in_cgray".

Reply:
The reason why it doesn't have to be labelled tht way in the pydicom elements is that the definition of which units are used is in the Standard.